ptbcontrib/postgres_persistence/postgrespersistence.py

```diff
@@ -89,8 +89,9 @@ def __init__(
         self.__load_database()
 
     def __init_database(self) -> None:
-        """creates table for persistence data if not exists"""
-
+        """
+        creates table for persistence data if not exists already.
+        """
```

Comment:
This doesn't sound much of a reformulation to me tbh :D

Reply:
Lol, I suck at writing docs